org.gnome.Epiphany.json

```diff
@@ -25,6 +25,18 @@
                     "commit": "5b7ffe85cded2fa3ce682510706601c1289da004"
                 }
             ],
+    "modules": [
+        {
+            "name": "gstreamer-vaapi",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/GStreamer/gstreamer-vaapi",
```

Comment:
Additionally, please use the mono repo https://gitlab.freedesktop.org/gstreamer/gstreamer/ and pick the gstreamer-vaapi submodule from there, see how Decoder for example does it

Reply:
oh thanks i will make it now